util/cli/cli.go

```diff
@@ -204,16 +204,21 @@ func setComments(input []byte, comments string) []byte {
 	return []byte(strings.Join(parts, "\n"))
 }
 
+// InteractiveEdit launches an interactive editor
 func InteractiveEdit(filePattern string, data []byte, save func(input []byte) error) {
 	editor := os.Getenv(editorEnv)
 	if editor == "" {
 		editor = defaultEditor
 	}
+	var editorArgs []string
+	if strings.Contains(editor, " ") {
+		editorArgs = strings.Split(editor, " ")
+	}
 	errorComment := ""
 	for {
 		data = setComments(data, errorComment)
 		tempFile := writeToTempFile(filePattern, data)
-		cmd := exec.Command(editor, tempFile)
+		cmd := exec.Command(editor, editorArgs[1], tempFile)
```

Comment:
If no editor is specified then CLI might crash here due to nil pointer dereference. Also editor might has more than one argument. I would propose change it as following:

```
	var editor string
	var editorArgs []string
	if overrideEditor := os.Getenv(editorEnv); overrideEditor == "" {
		editor = defaultEditor
	} else {
		parts := strings.Fields(overrideEditor)
		editor = parts[0]
		editorArgs = parts[1:]
	}

	errorComment := ""
	for {
		data = setComments(data, errorComment)
		tempFile := writeToTempFile(filePattern, data)
		cmd := exec.Command(editor, append(editorArgs, tempFile)...)
```

@cabrinha let me know what you think please

Reply:
I really like this solution.